app/data_models/progress_store.py

```diff
@@ -134,7 +144,7 @@ def get_section_status(
 
     def get_completed_block_ids(
         self, section_id: str, list_item_id: Optional[str] = None
-    ) -> List[Optional[str]]:
+    ) -> Union[list[Optional[str]], Any]:
```

Comment:
Always a list of strings?

Reply:
'block_ids` are `List[Optional[str]]` in `Progress` class hence Any as return.

Reply:
Still outstanding, it should be ``list[str]` instead of list of optional strings.

Reply:
Sorted in another thread now!